### DISCUSSION TITLE
Possible bug on the "Repositories traffic" plugin

### DISCUSSION THREAD

**Discussion by eduardosantoshf** [lowlighter/metrics#1331] [Q&A] [answered]
Hi!

I've noticed the [Repositories traffic](https://github.com/lowlighter/metrics/blob/master/source/plugins/traffic/README.md) plugin is being rendered, even though I don't have that option on my workflow, and the default value is _no_. Is this a bug, or am I missing something? 

P.S. I've also set the value on the workflow to its default, but nothing changed:
```yaml
plugin_traffic: no
```

Thanks in advance for the answer :)

**Edit**: I realized that the repositories stats could be hidden using the _base_ option found [here](https://github.com/lowlighter/metrics/blob/master/source/plugins/base/README.md). So I guess the "problem" is solved :)

**Comment by lowlighter** [comment, marked as answer]
Hi!

Yeah the `base` plugin is kind of a legacy plugin, that's why it differs a bit from how other plugins work 😅 
Glad you found the answer though !

Thanks a lot for using this app 😄 !
